CONTRIBUTING.md

```diff
@@ -1,6 +1,13 @@
 # Primer
 
-ZHA device handlers and it's provided Quirks allow Zigpy, ZHA and Home Assistant to work with non standard Zigbee devices. If you are reading this you may have a device that isn't working as expected. This can be the case for a number of reasons but in this guide we will cover the cases where functionality is provided by a device in a non specification compliant manner by the device manufacturer.
+ZHA device handlers and its provided "quirks" allow the zigpy library and applications implementing it, like the ZHA integration implementation in Home Assistant, to work with non-standard Zigbee devices. And just to clearify, this ZHA device handlers library and its "quirks" has no dependencies on the ZHA integration implementation in Home Assistant.
+
+If you are reading this you may have a Zigbee device that isn not working as expected. The reason for that being so can be a number of things and this guide we will only cover the cases where functionality is provided by a device in a non-specification compliant manner by the device manufacturer, meaning that the manufacturer did not quite fully follow the Zigbee specification. 
```

Comment:
Isn -> is

Reply:
The sentence at the end doesn’t flow as well as the original that you replaced. I’d put the original sentence back.

 “ This can be the case for a number of reasons but in this guide we will cover the cases where functionality is provided by a device in a non specification compliant manner by the device manufacturer.”

Reads much better IMO

Reply:
still have the first typo mentioned above: "If you are reading this you may have a Zigbee device that `isn` not"

Reply:
OK update CONTRIBUTING.md again with a rephrasing of the introduction based on you feedback.